sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_v1/query.sql

```diff
@@ -86,6 +90,81 @@ WITH combined AS (
     AS provider,
   FROM
     contextual_services.topsites_click
+  UNION ALL
+  SELECT
+    metrics.uuid.top_sites_context_id AS context_id,
+    DATE(submission_timestamp) AS submission_date,
+    'topsites' AS source,
+    IF(events[SAFE_OFFSET(0)].name = 'contile_click', 'click', 'impression') AS event_type,
+    'mobile' AS form_factor,
+    normalized_country_code AS country,
+    metadata.geo.subdivision1 AS subdivision1,
+    metrics.string.top_sites_contile_advertiser AS advertiser,
+    'release' AS release_channel,
+    CAST(
+      (SELECT value FROM UNNEST(events[SAFE_OFFSET(0)].extra) WHERE key = 'position') AS INT64
+    ) AS position,
+    CASE
+    WHEN
+      metrics.url2.top_sites_contile_reporting_url IS NULL
+    THEN
+      'remote settings'
+    ELSE
+      'contile'
+    END
```

Comment:
```suggestion
    'contile'
```

Do we need the conditional? Or are we guaranteed that telemetry is only sent from mobile for tiles served from contile?